src/posts/2022-01-31-liz-on-infoq/index.md

```diff
@@ -0,0 +1,13 @@
+---
+date: '2022-01-31T17:00:00.000Z'
+title: 'Liz Rice on Programming the Linux Kernel with eBPF, Cilium and Service Meshes'
+ogImageUrl: 'https://imgopt.infoq.com/fit-in/297x215/filters:quality(80)/podcasts/architecture-design-trends-report-2021/en/mediumimage/InfoQ-Podcast-290x210-1640630697522.jpg'
```

Comment:
Better replace with a larger version like https://www.infoq.com/styles/static/images/podcast/podcast-infoq.png

Reply:
Thanks, that is what I was looking for but couldn't find